packages/edge-login-ui-rn/src/native/components/common/HeaderBackButton.js

```diff
@@ -19,9 +19,7 @@ class HeaderBackButton extends Component<Props> {
     return (
       <TouchableOpacity style={styles.backButton} onPress={this.props.onPress}>
         {withArrow && <Icon name={icon} style={styles.backIconStyle} />}
-        {withArrow && !isIos ? null : (
-          <Text style={[styles.sideText]}>{this.props.label}</Text>
-        )}
+        {withArrow && <Text style={[styles.sideText]}>{this.props.label}</Text>}
```

Comment:
@jamesrisberg Unfortunately you will still want to check if it's iOS or Android because @paullinator specifically wants the header icons to mimic the operating system header. That's why we have this conditional checking for operating system. We do not want completely parity.